dashboard/app/assets/stylesheets/application.scss

```diff
@@ -2676,7 +2676,7 @@ a.download-video {
   }
 }
 
-.text-match, .free-response, .peer-review {
+.text-match, .peer-review {
```

Comment:
have you looked at a peer-review level to see if the styles are messed up there too?